compiler/src/iree/compiler/Reducer/Framework/README.md

```diff
@@ -0,0 +1,8 @@
+# TODO
```

Comment:
readmes as todos bitrot fast - keep these in issues or add more permanent documentation (what "framework" is, etc)

Reply:
Currently removing the TODOs and will send them as issues instead.